packages/frontend-v2/utils/plan/updatePlanOnDragEnd.ts

```diff
@@ -52,7 +52,8 @@ export const updatePlanOnDragEnd = (
       }
 
       if (oldTerm === newTerm) {
-        throw new Error("Course is being dragged over its own term");
+        // Course is being dragged over its own term, so we do nothing
+        return;
```

Comment:
any reason we removed this error? the way updatePlanOnDragEnd is used in home is that we continue with the update if the function doesn't throw an error, don't see why we need to continue with the update if the course is being dragged over its own term

Reply:
Ah I was confused why we were throwing for a non-fatal error that we handled, but that if it's used in control flow above this function, that makes sense. I'll revert that change and leave a comment in the function description.